taskt/Core/Automation/Commands/Data Commands/FormatDataCommand.cs

```diff
@@ -1,77 +1,74 @@
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to apply formatting to a string")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to apply specific formatting to text or a variable")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements actions against VariableList from the scripting engine.")]
+    [Group("Data Commands")]
+    [Description("This command converts a text (either a date or a number) to a specified format and saves result in a variable.")]
     public class FormatDataCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please supply the value or variable (ex. [DateTime.Now]")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Specify either text or a variable that contains a date or number requiring formatting")]
-        [Attributes.PropertyAttributes.SampleUsage("[DateTime.Now], 1/1/2000, 2500")]
-        [Attributes.PropertyAttributes.Remarks("You can use known text or variables.")]
-        public string v_InputValue { get; set; }
+        [PropertyDescription("Input Data")]
+        [InputSpecification("Specify either text or a variable that contains a date or number requiring formatting.")]
+        [SampleUsage("1/1/2000 || 2500 || {DateTime.Now} || {vNumber}")]
+        [Remarks("You can use known text or variables.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_InputData { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select the type of data")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Date")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Number")]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate the source type")]
-        [Attributes.PropertyAttributes.SampleUsage("Choose **Date** or **Number**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Input Data Type")]
+        [PropertyUISelectionOption("Date")]
+        [PropertyUISelectionOption("Number")]
+        [InputSpecification("Select the type of input data.")]
+        [SampleUsage("")]
+        [Remarks("Select 'Date' if input data is Date or 'Number' if it is a Number. Input data of other types will cause error.")]
```

Comment:
Change to "Select 'Date' if the input data is a Date or 'Number'..." and "other types will result in an error."

Reply:
done